css/mobile.css

```diff
@@ -1,425 +1,1208 @@
+    background-repeat: no-repeat;
+    background-size: auto;
+    background-position: right bottom;
+    margin-block: 0;
+    row-gap: 0;
+    display: flex;
+    flex-direction: column;
+    justify-content: left;
+    list-style-type: disc;
+    list-style-position: inside;
+  }
+
+  .grid-item3 {
+    grid-column: 1;
+    grid-row: 2;
+    color: #fff;
+    font-size: 0.8em;
+    height: 20vw;
+    width: 20vw;
+    background-color: #febf52;
+    background-image: url("skills.png");
+    background-repeat: no-repeat;
+    background-size: auto;
+    background-position: right bottom;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    list-style-type: disc;
+    list-style-position: inside;
+    margin-block: 0;
+    row-gap: 0;
+  }
+
+  .grid-item4 {
+    display: grid;
+    grid-column: 2;
+    grid-row: 2;
+    height: 20vw;
+    width: 20vw;
+    padding: 10vw 0 0 5vw;
+    background-image: url("/css/img-desktop-grid.png");
+    background-repeat: no-repeat;
+    background-size: auto;
+    background-position: 110% 30%;
+  }
+
+  fieldset.contact {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    height: 100%;
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    margin-block: 0;
+    box-sizing: border-box;
+    background-image: url("/css/formimg1.png"), url("/css/formimg2.png");
+    background-repeat: no-repeat, no-repeat;
+    background-position: left top, right bottom;
+    background-size: 20%, 20%;
+    background-color: #f5f5f5;
+  }
+
+  .contactheader {
+    padding: 0;
+    margin: 0;
+    font-size: 5.5em;
+    width: 40vw;
+    text-align: center;
+    display: flex;
+    flex-wrap: nowrap;
+  }
+
+  .name {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    height: 100%;
+    width: 19vw;
+    margin: 0 1%;
+    padding: 6%;
+    box-sizing: border-box;
+  }
+
+  .form-group {
+    font-family: "poppins", sans-serif;
+    display: inline-flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-evenly;
+    width: 40%;
+    margin: 1%;
+    padding: 1%;
+    box-sizing: border-box;
+    column-gap: 2%;
+    text-align: left; /* row-gap: 2%; */
+  }
+
+  .message-label {
+    font-family: "poppins", sans-serif;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: start;
+    width: 40%;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+
+  .message {
+    display: flex;
+    justify-content: center;
+    width: 40%;
+    min-height: 18%;
+    margin-bottom: 2%;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: "Poppins", sans-serif;
+  }
+
+  footer {
+    display: flex;
+    flex-direction: row;
+    align-items: end;
+    padding-bottom: 0;
+    font-size: 0.5em;
+    width: 100%;
+    height: 10vh;
+    position: relative;
+  }

```

Comment:
It would be nice to make your `footer` look like that of Figmas

## Your Design

![image](https://user-images.githubusercontent.com/77106962/144726001-5848dc1e-723d-4f16-ac12-13a8a8cb6cd0.png)

## Figma's Design

![image](https://user-images.githubusercontent.com/77106962/144726012-2eecb854-beaf-4752-9d81-10ea29303b00.png)

Reply:
Thanks is done.